node-notes-cli/app.js

```diff
@@ -0,0 +1,95 @@
+const fs = require('fs');
+
+const keyWord = process.argv[2];
+const keyWordLib = ['read', 'create', 'update', 'delete'];
+
+if (!keyWordLib.includes(keyWord)) {
+  console.error({ error: 'invalid command' });
+} else if (keyWordLib.includes(keyWord)) {
+  if (keyWord === 'read') {
+    fs.readFile('data.json', 'utf8', (err, data) => {
+      if (err) {
+        console.error(err);
+      }
+      const notesObj = JSON.parse(data);
+      for (const notesId in notesObj.notes) {
+        console.log(notesId + ': ' + notesObj.notes[notesId]);
+      }
+    });
+  } else if (keyWord === 'create') {
+    fs.readFile('data.json', 'utf8', (err, data) => {
+      if (err) {
+        console.error(err);
+      }
+      const notesObj = JSON.parse(data);
+      const newId = notesObj.nextId;
+      if (!process.argv[3]) {
+        console.error({ error: 'invalid argument. please input a string' });
+        return;
+      }
+      notesObj.notes[newId] = process.argv[3];
+      notesObj.nextId++;
+      console.log(`Note ID ${newId} successfully created`);
+      fs.writeFile('data.json', JSON.stringify(notesObj, null, 2), 'utf8', err => {
+        if (err) {
+          console.error(err);
+        }
+      });
```

Comment:
Technically, a new note isn't successfully created until its persisted to the data file, so I'd recommend waiting until that finishes successfully before letting the user know.